apps/web/components/booking/BookingDescription.tsx

```diff
@@ -102,6 +106,28 @@ const BookingDescription: FC<Props> = (props) => {
             </div>
           </div>
         )}
+        {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
+        {/* @ts-ignore */}
```

Comment:
The type definition for `eventType` is really complex, Please do help If anyone knows how to deal with it.